### DISCUSSION TITLE
Bulk Delete on Foods, Units, Labels on Manage Data Page

### DISCUSSION THREAD

**Discussion by Kuchenpirat** [mealie-recipes/mealie#2729] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Mealie has the great Manage Data Page and recipes already have a lot of bulk actions. 

Foods, Units and Labels do not yet have that option.

### Please provide a concise description of the feature that would resolve your issue.

Use the existing bulkActions infrastructure of the crudTable to bulk delete selected foods, units, labels. 
This will unify the feel of the manage Data page and allow users to easily delete multiple items at once. 
Especially practical if you have you have seeded your database and want to clean up the seeded data. 

### Please consider and list out some caveats or tradeoffs made in your design decision

Can't think of any. 

Please give feedback. 

Will gladly work on this if it is accepted and there are no major cons. 

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by mhoenes** [comment]
One caveat is that you can't see if the food, unit, label is still referenced in any recipe.
This potentially leaves you with "broken" recipes, if you're not careful.

But, this caveat is not lmited to bulk deletions, but deletions in general.
